charts/camunda-platform/templates/NOTES.txt

```diff
@@ -67,7 +67,9 @@ The Cluster itself is not exposed as a service that means that you can use `kube
 
 Now you can connect your workers and clients to `localhost:26500`
 
+{{- if or (.Values.zeebe.enabled) (.Values.operate.enabled) (.Values.tasklist.enabled) (.Values.optimize.enabled) (.Values.identity.enabled) (.Values.elasticsearch.enabled) }}
 ### Connecting to Web apps
+{{- end }}
```

Comment:
@DiptoChakrabarty this will just hide the section title, it should be at the end of the whole section. (simply at the last line of the file).

Reply:
done